server/graphql/demographic/src/types.rs

```diff
@@ -121,56 +122,6 @@ impl DemographicProjectionConnector {
         }
     }
 }
-#[derive(PartialEq, Debug)]
```

Comment:
Moved the nodes to shared `types` crate.
Tried to move the minimum required, so connectors etc are still here.